### PR TITLE
Plugin updater

### PR DESCRIPTION
**Pull request by wpscholar**
Add support for our new plugin update API.

### CODE REVIEW

File: languages/mojo-marketplace-wp-plugin.pot [hunk 1]
```diff
@@ -3,8 +3,9 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: MOJO Marketplace 1.5.2\n"
-"Report-Msgid-Bugs-To: https://wordpress.org/support/plugin/mojo-plugin\n"
-"POT-Creation-Date: 2020-04-15 18:54:32+00:00\n"
+"Report-Msgid-Bugs-To: "
+"https://wordpress.org/support/plugin/mojo-marketplace-wp-plugin\n"
```
Comment (0aveRyan) [inline languages/mojo-marketplace-wp-plugin.pot:7]: This is autogenerated? Is there a flag we can set when we run the language pack translation to not have this point at WordPress.org?

File: mojo-marketplace.php [hunk 2]
```diff
@@ -27,16 +29,13 @@
...
  */
 function mojo_marketplace_load_plugin_textdomain() {
-	load_plugin_textdomain( 'mojo-marketplace-wp-plugin', FALSE, basename( dirname( __FILE__ ) ) . '/languages/' );
+	load_plugin_textdomain( 'mojo-marketplace-wp-plugin', false, basename( dirname( __FILE__ ) ) . '/languages/' );
 }
+
 add_action( 'plugins_loaded', 'mojo_marketplace_load_plugin_textdomain' );
 
 // Composer autoloader
-if ( version_compare( phpversion(), 5.3, '<' ) ) {
```
Comment (0aveRyan) [inline mojo-marketplace.php:35]: Can we remove this? We don't have a minimum PHP version set in the Plugin File Header above 🤔

Comment (wpscholar) [reply mojo-marketplace.php:35]: @0aveRyan This has been removed... ;) I'll add the Requires headers to the main plugin file though, as that info is used by our custom plugin update API.

File: languages/mojo-marketplace-wp-plugin.pot (hunk 1 above)
Comment (wpscholar) [reply languages/mojo-marketplace-wp-plugin.pot:7]: @0aveRyan It appears that there is no way to change this using our current tooling. It does look like there is a way to do it using WP CLI: https://github.com/wp-cli/i18n-command/issues/47

I'll look into the best way to handle this as I'm considering automating the generation of the `.pot` file.

**Comment by wpscholar** [comment]
@0aveRyan Ready for a final review. I've updated the plugin loading process to accommodate a PHP compatibility check.  I've also updated how the `.pot` file is generated to allow for more control over what gets generated.

**Pull request merged**